src/server/config/schema.js

```diff
@@ -162,6 +162,13 @@ module.exports = () => Joi.object({
   status: Joi.object({
     allowAnonymous: Joi.boolean().default(false)
   }).default(),
+  map: Joi.object({
+    manifestServiceUrl: Joi.when('$dev', {
+      is: true,
+      then: Joi.string().default('https://geo.elastic.co/v1/manifest'),
```

Comment:
yes, for now this needs to be the same. Infra has only setup a single production server. this will likely change in the weeks to come.